lit_nlp/examples/models/stanza_models.py

```diff
@@ -0,0 +1,107 @@
+# Lint as: python3
```

Comment:
add copywrite header to all files

Reply:
Added. I think I did it right, but let me know.